frontend/src/Comment.svelte

```diff
@@ -36,25 +36,27 @@
     }
 </script>
 
-<div class="comment comment-{unread ? 'unread' : 'read'} {type}" on:dblclick={() => editing = can_edit}>
-  <strong>{author}</strong>: 
-  {#if !editing}
-    {#if type == 'automated'}
-      {text}
-      {#if url}
-        <a href="{url}">
-          <span class="iconify" data-icon="entypo:help"></span>
-        </a>
+  <div style="display: flex; flex-direction: row;">
```

Comment:
bootstrap has some helper classes like `class="d-flex flex-row"`, but i think it is okay to have it like this... At least we can easily upgrade bootstrap version - they like changing names of these helpers :(

We could also use "component style tag" which affects only elements inside this single component:
```
<style>
.container {
   display: flex;
   flex-direction: row;
}
</style>

<div class="container">
   <!-- style affects only this one class in Comment.svelte -->
</div>
```
But I would use it if we have more css properties - your solution is simpler and easier so I would keep it

Reply:
OK, we can do it later on :-)